services/event-application-service/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rlevkovych/lohfindercpp:latest
```

Comment:
you have your own base image?

Reply:
Yes, I have pre-built libraries needed for the project. So, I will have smaller build time for my service.